creator/ingest_runs/signals.py

```diff
@@ -13,3 +16,41 @@ def ingest_run_pre_save(sender, instance, *args, **kwargs):
     if instance.versions.count() > 0:
         instance.name = instance.compute_name()
         instance.input_hash = instance.compute_input_hash()
+
+
+@receiver(post_transition, sender=IngestRun)
+def ingest_run_post_transition(sender, instance, name, source, target,
+                               *args, **kwargs):
+    """
+    Create a corresponding Event after the status of an IngestRun is changed.
+    """
+    if target == "running":
+        target = "started"
+    TARGET_EVENT_TYPES = {
+        "started": "IR_STA",
+        "complete": "IR_COM",
+        "canceled": "IR_CAN",
+        "failed": "IR_FAI",
+    }
+    versions = list(instance.versions.all())
+    username = instance.creator.display_name or "Anonymous user"
```

Comment:
This should be taken care of here: https://github.com/kids-first/kf-api-study-creator/blob/master/creator/models.py#L95